packages/paper/src/lib/Paper.tsx

```diff
@@ -1,7 +1,13 @@
 import { forwardRef } from 'react';
 import { Paper as MuiPaper, PaperProps as MuiPaperProps } from '@mui/material';
 
-export type PaperProps = Omit<MuiPaperProps, 'classes' | 'style' | 'className' | 'sx'>;
+export type PaperProps = Omit<MuiPaperProps, 'classes' | 'style' | 'className' | 'sx' | 'variant'> & {
+  /**
+   * The variant to use.
+   * @default 'outlined'
```

Comment:
I see we're not actually setting this as the default. Is this just for the storybook?

Reply:
Yup. Just for docs. It wouldn't let me set the default value in the types. I did use the theme though to set a default for the prop.

Reply:
ahhh, glad to see that's actually working. I think I ran into an issue with disableRipple not working as expected. Time to go back over that branch.